sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableAsyncClient.java

```diff
@@ -602,15 +697,22 @@ public String generateSas(TableSasSignatureValues tableSasSignatureValues) {
     }
 
     /**
-     * Lists entities using the parameters in the provided options.
+     * Lists {@link TableEntity entities} using the parameters in the provided options.
+     *
+     * If the {@code filter} parameter in the options is set, only {@link TableEntity entities} matching the filter
+     * will be returned. If the {@code select} parameter is set, only the properties included in the select parameter
+     * will be returned for each {@link TableEntity entity}. If the {@code top} parameter is set, the maximum number of
+     * returned {@link TableEntity entities} will be limited to that value.
```

Comment:
Top is semantically different than what is being mentioned here.

https://docs.microsoft.com/en-us/rest/api/storageservices/query-entities#remarks

> Note that the $top query option in this case specifies the maximum number of results per page, not the maximum number of results in the whole response set.

Reply:
Thank you, I'll make that clear.